### PR TITLE
Color from hex code

### PR DESCRIPTION
**Pull request by thled**
This is an addition to https://github.com/mjkillough/cnx/pull/49 so you can also create a color from a hex code.

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -0,0 +1 @@
+use nix
```
Comment (psibi) [inline .envrc:1]: Do you think .envrc can be added to gitignore and not part of this commit ?

The reason being different people might want use to use different `.envrc`.

**Review by psibi — APPROVED** [review]
LGTM apart from minor comment. Can you also update changelog ?

File: shell.nix [hunk 2]
```diff
@@ -2,20 +2,26 @@ with import <nixpkgs> { };
...
     llvmPackages.libclang
+    openssl
+    pango
+    pkg-config
+    python3
+    rust-bindgen
+    rustc
+    rustfmt
+    wirelesstools
+    xorg.libxcb
```
Comment (psibi) [inline shell.nix:24]: I guess this is getting repeated once more ?

File: .envrc (hunk 1 above)
Comment (thled) [reply .envrc]: Yes, that's reasonable.

File: shell.nix (hunk 2 above)
Comment (thled) [reply shell.nix:24]: Good catch

**Comment by psibi** [comment]
Thank you!

**Pull request merged**